### DISCUSSION TITLE
Sostituzione variabili nelle email

### DISCUSSION THREAD

**Discussion by dlondero** [fattureincloud/api#78] [Q&A] [answered]
Ciao, sto provando a schedulare delle email per l'invio dei documenti. L'email viene spedita correttamente ma la sostituzione delle variabili non viene eseguita. Di seguito un estratto del codice:

```php
$emailSchedule = new EmailSchedule();
$emailSchedule->setSubject('{{documento-numerato}} from {{ragione-sociale}}');
$emailSchedule->setBody('Dear {{cliente}},<br/>to see our {{documento}} of {{ammontare-lordo}} in PDF...');
$emailSchedule->setInclude($emailScheduleInclude);
...

$scheduleEmailRequest = new ScheduleEmailRequest();
$scheduleEmailRequest->setData($emailSchedule);

$issuedDocumentsApi->scheduleEmail($companyId, $documentId, $scheduleEmailRequest);
```

Nella mail ricevuta il subject è `{{documento-numerato}} from {{ragione-sociale}}` senza sostituzione variabili e lo stesso avviene per il body. Da notare che nel body `{{allegati}}` viene invece rimpiazzato con il button per scaricare l'allegato (come definito in `$emailScheduleInclude`).

Cosa mi sfugge?
Grazie in anticipo.

**Comment by dlondero** [comment]
Forse semplicemente non c'è possibilità di usare le variabili se non per `{{allegati}}`?

**Comment by emajo** [comment, marked as answer]
Buongiorno @dlondero, attualmente le uniche variabili che mettiamo a disposizione sono `{{allegati}}`, `{{logo-1}}` e `{{logo-2}}`.

Rimango a disposizioni per ulteriori chiarimenti,
Emanuele.

**Reply by dlondero** [reply]
Ottimo, grazie. Magari si può aggiungere l'informazione da qualche parte nella documentazione.
